### ISSUE TITLE
Add Library DTE to Destroy 

### ISSUE THREAD

**Issue by GoldenOre** [OvercastNetwork/Rotations#535]
Library is a pretty solid cannon based dte map. Ratings are 👌

https://oc.tc/forums/topics/5845d56667d522311800007b

**Comment by Scap298** [comment]
I think it will be cool to have some maps for the persons who like cannoning but have to remember that not everyone likes that or are nre players but the experienced never player should never be forget, so it will be cool like 1 or 2 maps like DATAStation or this one, that is just cannons

**Comment by mitchiiii** [comment]
The issue with DTE maps like this is that if you don't have a TNT license you basically can't do anything, and I'm pretty sure that was one of the reasons DATAstation was removed from rotation.

**Comment by Scap298** [comment]
Exactly, I remember when it got removed, and also for new players and people like me that dont like cannons, but I say to the people who like them, thats what I say at least 1 for them but that will be good for a discuss

Obtener Outlook para Android<https://aka.ms/ghei36>



On Sat, Dec 10, 2016 at 4:13 AM -0600, "Mitchell" <notifications@github.com<mailto:notifications@github.com>> wrote:


The issue with DTE maps like this is that if you don't have a TNT license you basically can't do anything, and I'm pretty sure that was one of the reasons DATAstation was removed from rotation.

-

**Comment by GoldenOre** [comment]
@mitchiiii You get a bow in the spawn kit to get kills. The map is also a lot more open than DataStation making it easier to shoot people. 

@Scap298 I originally thought that it would be hard for new players to play it but after watching play tests a few times its actually pretty easy for them to learn how to cannon. Since its a pretty big map and the bookcase(frame) can't be destroyed like DataStation players can easily play around with cannons/TNT. Although most new people just watch more experienced players build cannons then copy them.
